src/index.js

```diff
@@ -74,14 +77,28 @@ if (config.auth) {
   createDefaultTables();
 } else {
   console.warn(
-    "Warning: Soul is running in open mode without authentication or authorization for API endpoints. Please be aware that your API endpoints will not be secure.",
+    'Warning: Soul is running in open mode without authentication or authorization for API endpoints. Please be aware that your API endpoints will not be secure.',
   );
 }
 
-app.use("/api/docs", swaggerUi.serve, swaggerUi.setup(swaggerFile));
-app.use("/api", rootRoutes);
-app.use("/api/tables", tablesRoutes);
-app.use("/api/tables", rowsRoutes);
+//If the updateuser command is passed from the CLI execute the updateuser function
+if (argv._.includes('updateuser')) {
+  const { id, password, is_superuser } = argv;
+
```

Comment:
@AbegaM - we should also check that the id field is present

Hmm, should we also do `process.exit()` once we have finished updating the user?  (I don't think users will expect `soul` server to be running after doing `updateuser`.

Reply:
Yes, we will do `process.exit()` once the update is completed, and if you check the `cli.js` file i have made the `--id` field required in the schema so there would be no need to check it inside this function, the CLI library (`yargs`) will return an error